src/index.css

```diff
@@ -1,32 +1,49 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;800&family=VT323&display=swap');
 
 :root {
-  --primary: #12bca2;
-  --secondary: #6d15df;
+  --primary: #1f1f1f;
+  --secondary: #3D7EAA;
 }
-h1,h2,h3 {
+
+/* --secondary: linear-gradient(to bottom right, #f0c27b, #4b1248) */
+
+
+h1,
+h2,
+h3 {
   margin: 0;
 }
 
 body {
   margin: 0;
-  color: #333;
+  color: #FFF;
   font-family: 'Poppins';
-  background: #f2f2f2;
+  background: #121212;
 }
+
+img {
+  width: 100%;
+  object-fit: cover;
+  height: 300px;
+}
+
 nav {
   background: var(--primary);
   padding: 10px 20px;
   text-align: center;
+  margin: 1.5rem 0;
```

Comment:
There may be some exceptions, but prefer to always use the same units for everything. If you're going to use pixels, don't mix with em or rem, especially with margins. If the user increases the font size or zooms in, they can get all wonky.